scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -614,6 +645,70 @@ cdef _dijkstra_directed(
 
     free(nodes)
 
+cdef _dijkstra_directed_multi(
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] source_indices,
+            np.ndarray[DTYPE_t, ndim=1, mode='c'] csr_weights,
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] csr_indices,
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] csr_indptr,
+            np.ndarray[DTYPE_t, ndim=1, mode='c'] dist_matrix,
+            np.ndarray[ITYPE_t, ndim=1, mode='c'] pred,
+            DTYPE_t limit):
+    cdef unsigned int Nind = source_indices.shape[0]
+    cdef unsigned int N = dist_matrix.shape[0]
+    cdef unsigned int i, k, j_source, j_current
+    cdef ITYPE_t j
+
+    cdef DTYPE_t next_val
+
+    cdef int return_pred = (pred.size > 0)
+
```

Comment:
There are so many `cdef` lines that putting them all in a `cdef:` block might be preferable. Also, not sure if the occasional blank lines between `cdef`s are helpful or not.

Reply:
I submitted a commit that changes this style across the entire module, could be considered a different PR, but let me know what the standards are here.